### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

**Comment by chrmod** [comment]
@stefanpenner @joliss this PR broke master. Update of `broccoli-slow-trees` relies on `heimdalljs` to provide timing information. This results in this error:
```
Error when printing slow nodes: TypeError: tree.visitPostOrder is not a function
``` 
once trying to `broccoli serve` any build tree.

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/broccolijs/broccoli",
   "dependencies": {
     "broccoli-node-info": "1.1.0",
-    "broccoli-slow-trees": "^2.0.0",
+    "broccoli-slow-trees": "3.0.1",
```
Comment (Turbo87) [inline package.json]: @bekzod why did you pin this dependency instead of using a caret constraint?

Comment (bekzod) [reply package.json]: because I am crazy :D .... tbh I think it was accident

Comment (bekzod) [reply package.json]: sorry

**Pull request merged**